expression/builtin_time_vec.go

```diff
@@ -1352,11 +1352,59 @@ func (b *builtinWeekWithoutModeSig) vecEvalInt(input *chunk.Chunk, result *chunk
 }
 
 func (b *builtinUnixTimestampDecSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinUnixTimestampDecSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	result.ResizeDecimal(n, false)
+	Ts := result.Decimals()
+	timeBuf, err := b.bufAllocator.get(types.ETTimestamp, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(timeBuf)
+	if err := b.args[0].VecEvalTime(b.ctx, input, timeBuf); err != nil {
+		var row chunk.Row
```

Comment:
Why do we need this? How about returning this error directly?

Reply:
when an error happens, the non-vec function returns zero. So we have to  handle the error like here(link: https://github.com/pingcap/tidb/pull/13098#discussion_r347720141)

Reply:
Could we call `b.evalDecimal` here instead of copying and pasting?

Reply:
@SunRunAway ok, i will do it.